### ISSUE TITLE
Client Lite 1.0.0 Nordic NRF5840_DK+BG 96 combination not compiling against Mbed OS 6.0 Alpha 3

### ISSUE THREAD

**Issue by JanneKiiskila** [ARMmbed/pelion-client-lite-example#3]
Lite 1.0.0 example compilation fails against Mbed OS 6.0.0 Alpha 3.

```
Building project pelion-client-lite-example-sources-internal (NRF52840_DK, GCC_ARM)
Scan: pelion-client-lite-example-sources-internal
[ERROR] Attempt to override undefined parameter 'ppp-cell-iface.baud-rate' in 'application[NRF52840_DK]'
[mbed] ERROR: "/home/jankii01/venv/mbed/bin/python" returned error.
       Code: 1
       Path: "/home/jankii01/mbed/pelion-client-lite-example-sources-internal"
       Command: "/home/jankii01/venv/mbed/bin/python -u /home/jankii01/mbed/pelion-client-lite-example-sources-internal/mbed-os/tools/make.py -t GCC_ARM -m NRF52840_DK --profile profiles/pico_lte_size.json --profile mbed-os/tools/profiles/extensions/minimal-printf.json --source . --build ./BUILD/NRF52840_DK/GCC_ARM-PICO_LTE_SIZE --stats-depth 13"
```

Reason being the removal of the ppp-cell-iface.baud-rate parameter in Mbed OS 6.0.

**Fix:**
- Parameter `"ppp-cell-iface.baud-rate"` can be removed completely from the `mbed_app.json`.
- One also has to remove the parameter `"ppp-cell-iface.apn-lookup"`.

**NOTE:** This will be fixed out of box in the next upcoming release.

**Comment by JanneKiiskila** [comment]
Rebased, force pushed.

**Comment by ciarmcom** [comment]
ARM Internal Ref: IOTCLT-4141

**Comment by JanneKiiskila** [comment]
This will be resolved to Client Lite 1.1.0 release (fixes done already).

**Comment by teetak01** [comment]
Fixed in #4
